ElectronClient/app/app.js

```diff
@@ -1037,6 +1069,13 @@ class Application extends BaseApplication {
 			css: cssString
 		});
 
+		const templates = await this.loadTemplates(Setting.value('profileDir') + '/templates');
+
+		this.store().dispatch({
+			type: 'LOAD_TEMPLATES',
```

Comment:
The action type format is "TYPE_VERB" (type is singular), for example "NOTE_SELECT". So in your case it would be "TEMPLATE_UPDATE_ALL" for consistency with the other actions.

Reply:
Thanks for the direction I've changed it to TEMPLATE_UPDATE_ALL